woodwork/table_accessor.py

```diff
@@ -1096,6 +1096,28 @@ def infer_temporal_frequencies(self, temporal_columns=None):
             (dict): A dictionary where each key is a temporal column from the DataFrame, and the
                 value is its observation frequency represented as a pandas offset alias string (D, M, Y, etc.)
                 or None if no uniform frequency was present in the data.
+
+        Note:
+            The pandas util ``pd.infer_freq``, which is used in this method, has the following behaviors:
+                - If even one row in a column does not follow the frequency seen in the remaining columns,
+                    no frequency will be able to be inferred. Example of otherwise daily data that skips one day:
```

Comment:
Maybe just say `no frequency will be inferred.` here and remove the `able to be` phrase?